user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/download/ProteomicsMeasurementContentProvider.java

```diff
@@ -31,11 +31,13 @@
  */
 public class ProteomicsMeasurementContentProvider implements DownloadContentProvider {
 
-  private static final String FILE_NAME = "proteomics_measurements.xlsx";
+  private static final String FILE_NAME_SUFFIX = "proteomics_measurements.xlsx";
```

Comment:
you could introduce this change for NGSMeasurementContentProvider as well

Reply:
done :)